packages/forma-36-react-components/src/components/CheckboxField/README.mdx

```diff
@@ -0,0 +1,57 @@
+# CheckboxField
+
+CheckboxField is a component that gives an possiblity to easy setup proper checkbox with its label. Component also contains an options that allows setting helper text displayed below the label. You might see checkboxes in forms, they might be also used for filtering data on the page or terms and conditions. Everywhere where user have multiple choice you might want to use this component.
+
+## Example of usage
+
+```jsx
+import {CheckboxField} from '@contentful/forma-36-react-components';
+
+<CheckboxField
+  checked={}
+  onChange={hadnleOnChange}
+  name="checkbox name"
+  id="randomId"
+  labelText="Label text"
+  helpText="Helper text displayed below"
+/>
+```
+
+Light label example
+
+```jsx
+import {CheckboxField} from '@contentful/forma-36-react-components';
+
+<CheckboxField
+  checked={}
+  onChange={hadnleOnChange}
+  name="checkbox name"
+  id="randomId"
+  labelText="Label text"
+  helpText="Helper text displayed below"
+  labelIsLight
+/>
+```
+
+## Content recommendations
+
+ - Start every option from a capital letter
+ - Do not use commas at the end of each line
+ - If the checkbox is used to agree to terms and conditions, we recommend to use first person in the sentence
+
+## Best practices
+
+ - Checkboxes should work independently, checking one should not affect any other checkbox in the interface. 
```

Comment:
I suggest linking the Radio component here since it's the proper component for lists where the user can choose only one option

Reply:
@gui-santos yeah, sounds good, but I think we don't have a README for radio component yet - which probably would be ideal. Maybe I will update as soon as we create docs for it?

Reply:
your call

Reply:
Done. I linked to the tsx for now. I will change it as soon as we have docs for RadioButton. ✅